apps/emqx_schema_registry/src/emqx_schema_registry_schema.erl

```diff
@@ -65,6 +95,42 @@ fields(json) ->
         {type, mk(json, #{required => true, desc => ?DESC("schema_type_json")})}
         | common_fields(emqx_schema:json_binary())
     ];
+fields(confluent_schema_registry) ->
+    [
+        {type,
+            mk(confluent, #{default => confluent, desc => ?DESC("schema_registry_external_type")})},
+        {url, mk(binary(), #{required => true, desc => ?DESC("confluent_schema_registry_url")})},
+        {auth,
+            mk(
+                hoconsc:union([none, ref(confluent_schema_registry_auth_basic)]),
+                #{default => none, desc => ?DESC("confluent_schema_registry_auth")}
+            )}
+    ];
+fields(confluent_schema_registry_auth_basic) ->
+    [
+        {mechanism,
```

Comment:
`basic` is the only option for now. so it can be hidden.

Reply:
hidden in the documentation sense?

Reply:
yes, IMPROTANCE_HIDDEN

Reply:
:checkered_flag: